pkgs/applications/misc/quicksynergy/default.nix

```diff
@@ -0,0 +1,36 @@
+{stdenv, fetchurl, pkgconfig, gtk2, synergy}:
+let
+  version = "0.9.0";
+in
+stdenv.mkDerivation {
```

Comment:
Good tip, thank you.

btw, why inherit is used in the second case, is it required or without 'inherit' the case would be sufficient?